buildenv/jenkins/variables/defaults.yml

```diff
@@ -295,9 +301,9 @@ x86-64_linux:
     all: '--enable-jitserver'
   build_env:
     cmd:
-      17: 'source /home/jenkins/set_gcc7.5.0_env'
-      18: 'source /home/jenkins/set_gcc7.5.0_env'
-      next: 'source /home/jenkins/set_gcc7.5.0_env'
+      all: 'source /home/jenkins/set_gcc7.5.0_env'
```

Comment:
We weren't using `source ...` for 8, 11 previously. We'll need to confirm it works to do so.

Reply:
fixed

Reply:
I didn't mean it needs to be changed, just that we need to run PR builds to confirm.